src/Datadog.Trace.ClrProfiler.Managed/Integrations/AdoNet/SqlCommandIntegration.cs

```diff
@@ -13,6 +13,7 @@ namespace Datadog.Trace.ClrProfiler.Integrations.AdoNet
     /// </summary>
     public static class SqlCommandIntegration
     {
+        private const string Major2 = "2";
         private const string Major4 = "4";
         private const string Major5 = "4";
```

Comment:
Not necessary to fix in this PR, but this looks like an oversight:
```suggestion
        private const string Major5 = "5";
```

Reply:
I'd also like to do a follow-up PR for this since it is unrelated to the minimum version change